manifests/params.pp

```diff
@@ -99,8 +99,6 @@
 
       if $postgresql::globals::postgis_package_name {
         $postgis_package_name = $postgresql::globals::postgis_package_name
-      } elsif $facts['os']['release']['major'] == '5' {
```

Comment:
Perhaps change this line to be `elsif $facts['os']['name'] == 'Fedora'` since that still uses the postgis package name.

Reply:
done